integrations/server/test_api_keys.py

```diff
@@ -0,0 +1,243 @@
+"""Integration tests for the API Keys"""
+import requests
+
+# first party
+from delphi.epidata.common.integration_test_base_class import BasicIntegrationTest
+
+
+class APIKeysTets(BasicIntegrationTest):
+    """Tests the API Keys behaviour"""
+
+    def setUp(self):
+        """Perform per-test setup."""
+
+        self.role_name = "cdc"
+        super().setUp()
+
+    def _make_request(self, url: str = None, params: dict = {}, auth: tuple = None):
+        if not url:
+            url = self.epidata.BASE_URL
+        response = requests.get(url, params=params, auth=auth)
+        return response
+ 
+    def test_public_route(self):
+        """Test public route"""
+        public_route = "http://delphi_web_epidata/epidata/version"
+        status_codes = set()
+        for _ in range(10):
+            status_codes.add(self._make_request(public_route).status_code)
+        self.assertEqual(len(status_codes), 1)
+        self.assertEqual(next(iter(status_codes)), 200)
```

Comment:
```suggestion
        self.assertEqual(status_codes, {200})
```
this is a little more concise.  the `next(iter(...` syntax seems a little weird to me ¯\\\_(ツ)_/¯ 

(there are some similar usages of this below too)

Reply:
Nice comment. I believe that I had used this `next(iter())` construction in order to get first element of the set and getting element by index is not applicable for `set` data type. Don't unrderstand why I didn't try to cast it to list ¯\_(ツ)_/¯, but your solution looks even better.